bamboo/integration_tests/accuracy_tests/accuracy_test.sh

```diff
@@ -10,21 +10,23 @@ FULL_LOG=0
 NODECOUNT=4
 PROCSPERNODE=2
 LBANN_DIR=$(git rev-parse --show-toplevel)
+EXECUTABLE="${LBANN_DIR}/build/${HOSTNAME}.llnl.gov/model_zoo/lbann"
```

Comment:
We should change this to use LBANN_EXE

Reply:
agreed. can you also change the default NODECOUNT variable to 1 or 2. 4 is excessive